front/src/pages/companies/constants/companyShowFieldDefinition.tsx

```diff
@@ -66,4 +69,23 @@ export const companyShowFieldDefinition: FieldDefinition<FieldMetadata>[] = [
       fieldName: 'createdAt',
     },
   } satisfies FieldDefinition<FieldDateMetadata>,
+  {
+    id: 'idealCustomerProfile',
+    label: 'ICP',
+    icon: <IconTarget />,
+    type: 'boolean',
+    metadata: {
+      fieldName: 'idealCustomerProfile',
+    },
+  } satisfies FieldDefinition<FieldBooleanMetadata>,
+  {
+    id: 'xUrl',
+    label: 'Twitter',
+    icon: <IconBrandX />,
+    type: 'url',
+    metadata: {
+      fieldName: 'xUrl',
+      placeHolder: 'X',
+    },
+  } satisfies FieldDefinition<FieldURLMetadata>,
```

Comment:
we should also add ARR

Reply:
We'll need a money editable field for that I think, with currency and such, I was planning to do it in a different PR for that reason but up to you! @charlesBochet